docs/reference/language-reference/built-in-types/bool.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "Bool"
+sidebar_label: "Bool"
+sidebar_position: 1
+keywords:
+  - boolean
+  - logic
+draft: false
+---
+
+Booleans are either `true` or `false`.
+
+```darklang
+let succeeded = true
+```
+
+## Methods
+
+### And
+
+The logical AND of two boolean values. Returns a bool.
```

Comment:
A truth table might be cool

Reply:
I'll add both a classic And truth table and an example that produces the same truth table.